Public/Src/App/Bxl/ServerDeployment.cs

```diff
@@ -257,48 +238,18 @@ private static Possible<Unit> TryKillProcess(string assemblyFullPath)
         }
 
         /// <summary>
-        /// Poisons the server deployment to cause it to be redeployed
+        /// Returns a calculated hash of the content of the BuildXL binaries deployment server cache directory.

```

Comment:
something is wrong in this sentence.  Maybe you meant
```suggestion
        /// Calculates a hash of the content of the BuildXL binaries from the server deployment directory.
```

Reply:
It's the balkan way of saying what you suggested, come on! 💃